Makefile

```diff
@@ -1,46 +1,20 @@
 .DEFAULT_GOAL:=help
 
-# define the name of the virtual environment directory
-VENV := .venv3.8.16
-
-# define the profile used by the dbt
-PROFILE := dwx_endpoint
-
 CHANGED_FILES := $(shell git ls-files --modified --other --exclude-standard)
 CHANGED_FILES_IN_BRANCH := $(shell git diff --name-only $(shell git merge-base origin/main HEAD))
 
 .PHONY: all install_deps dev_setup functional_test test clean help
 .PHONY: pre-commit pre-commit-in-branch pre-commit-all
 
-all: dev_setup test  ## Default target for dev setup and run tests.
-
-# Required python3 already installed in the system
-$(VENV)/bin/activate:
-	@\
-	python3 -m venv $(VENV)
-
-install_deps: $(VENV)/bin/activate dev-requirements.txt	## Install all dependencies.
-	@\
-	$(VENV)/bin/python3 -m pip install --upgrade pip
-	$(VENV)/bin/pip install -e . -r dev-requirements.txt
-
-dev_setup: $(VENV)/bin/activate	 ## Install all dependencies and setup pre-commit.
```

Comment:
@vamshikolanu why we need to remove the venv environment setup? 
How do we install python modules? Earlier the make dev_setup ensure the setup of venv and install required python modules. We need to manually install the dependencies like nox too !!

Reply:
Removed dependency of nox and reverted dev_setup steps.